setup.sh

```diff
@@ -4,21 +4,27 @@ jarvispath=$PWD
 touch jarvis
 chmod +rwx jarvis
 exec 3<> jarvis
-
     echo "#!/bin/bash" >&3
     echo "source $jarvispath/env/bin/activate" >&3
     echo "python $jarvispath/jarviscli/" >&3
-
 exec 3>&-
-
 sudo mv jarvis /usr/local/bin/jarvis
 
 UNAME=$(uname -s)
 if [[ "$UNAME" == "Darwin" ]]; then
   brew install ffmpeg
   brew install openssl
-  virtualenv env
+  read -n 1 -p "Specify python version(2/3)(Default-3)" answer
+  case ${answer:0:1} in
+    2 )
+        virtualenv env --python=python2.7
+    ;;
+    * )
+        virtualenv env --python=python3.6
+    ;;
+  esac
   . env/bin/activate
+  python --version
   pip install -r requirements.txt
```

Comment:
if python3 is selected we should use pip3

Reply:
same goes for pip. pip downloads for default python. Since default python is selected by virtual environment pip points to that version

Reply:
It doesn't work for me (MacOSX).

Reply:
Sure I will update 👍